server/index.js

```diff
@@ -36,6 +36,7 @@ app.get('/products/:product_id', (req, res) => {
       res.status(200).send(response.data);
     })
     .catch(error => {
+      console.log(error);
```

Comment:
The only note I have is this this console.log but other than that lgtm

Reply:
ty! will change that